src/main/java/jp/skypencil/errorprone/slf4j/Slf4jDoNotLogMessageOfExceptionExplicitly.java

```diff
@@ -49,11 +50,8 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
             .filter(meth -> THROWABLE.test(meth.sym.owner.toString()))
             .findFirst();
     if (problem.isPresent()) {
-      return Description.builder(
-              tree,
-              "Slf4jDoNotLogMessageOfExceptionExplicitly",
-              "https://github.com/KengoTODA/findbugs-slf4j#slf4j_manually_provided_message",
-              ERROR,
+      return buildDescription(tree)
```

Comment:
`BugPattern.buildDescription` builds the description with defaults from the `@BugPattern` annotation